tests.js

```diff
@@ -1177,28 +1233,32 @@ window.Specs = {
 			"overflow-block": {
 				"links": {
 					"tr": "#mf-overflow-block",
-					"dev": "#mf-overflow-block"
+					"dev": "#mf-overflow-block",
+					"mdn": "@media/overflow-block"
 				},
 				"tests": ["(overflow-block: none)", "(overflow-block: scroll)", "(overflow-block: optional-paged)", "(overflow-block: paged)"]
 			},
 			"overflow-inline": {
 				"links": {
 					"tr": "#mf-overflow-inline",
-					"dev": "#mf-overflow-inline"
+					"dev": "#mf-overflow-inline",
+					"mdn": "@media/overflow-inline"
 				},
 				"tests": ["(overflow-inline: none)", "(overflow-inline: scroll)"]
 			},
 			"color-gamut": {
 				"links": {
 					"tr": "#color-gamut",
-					"dev": "#color-gamut"
+					"dev": "#color-gamut",
+					"mdn": "@media/color-gamut"
```

Comment:
Looks like all media queries have the same format, so they can be generated too. Remember we know whether something is a media query when we generate the link.